pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -8,16 +8,33 @@ import (
 	"fmt"
 	"io/fs"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"runtime"
+	"sync"
 
 	"github.com/lima-vm/lima/pkg/debugutil"
 	"github.com/lima-vm/lima/pkg/limayaml"
 	"github.com/sirupsen/logrus"
 )
 
+// executableViaArgs0 returns the absolute path to the executable used to start this process.
+// It will also append the file extension on Windows, if necessary.
+// This function is different from os.Executable(), which will use /proc/self/exe on Linux
+// and therefore will resolve any symlink used to locate the executable. This function will
+// return the symlink instead.
+var executableViaArgs0 = sync.OnceValues(func() (string, error) {
+	executable, err := exec.LookPath(os.Args[0])
+	if err != nil {
+		return "", err
+	}
+	// LookPath() will add the `.exe` file extension on Windows, but will not return an
+	// absolute path if the argument contained any of `:/\` (or just `/` on Unix).
+	return filepath.Abs(executable)
+})
+
 func Dir() (string, error) {
-	self, err := os.Executable()
+	self, err := executableViaArgs0()
```

Comment:
os.ReadLink below should be removed

(failed to sleep after all 😅 )

Reply:
Ah, this is dead code because we call `os.Stat`, which will follow the symlink, so the returned mode bits never have the symlink bit set. The code would need to call `os.Lstat()` to actually work... Anyways, will remove it.